packages/example/gatsby-config.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
       resolve: 'gatsby-theme-carbon',
       options: {
         isSearchEnabled: true,
+        hasHeaderNavigation: false,
```

Comment:
Try to avoid boolean options for things that might have more than one conflicting styles/options. We want to make it clear that having header navigation means you _don't_ have sidebar navigation.

```suggestion
        navigationStyle: 'header',
```

Reply:
Ohh okay that's a good point - what would we use here instead? Or does it makes sense to update the variable name so it is a bit clearer?

Reply:
would we then create an option where they pass in 'sidebar' if they want the left nav? or would we just leave it blank? @vpicone

Reply:
@andreancardona yeah that would be the default option. Leaving it blank or providing it explicitly would do the same thing